abstract/myst.yml

```
@@ -1,35 +1,30 @@
 version: 1
 project:
-  # id: ubcgif-YEAR-AUTHORS-CONFERENCE
-  # title: TITLE
-  # description: DESCRIPTION
+  id: ubcgif-2023-kuttai-etal-agu
+  title: Comparing strategies for assessing uncertainty with geophysical inversions for mineral exploration
+  description: Talk presented in session SY41A-1036
```

```suggestion
  description: Talk presented at the American Geophysical Union (AGU) Fall Meeting in session SY41A-1036
```

@tetikshajain: for the AGU conferences, could you please also add the conference name (as per my suggestion above) in addition to the session number to the description? Thanks!

---

I accidentally made another backup branch while uploading. that branch can be deleted.